spec/helper.js

```diff
@@ -387,6 +394,11 @@ function mockShortLivedAuth() {
   return auth;
 }
 
+// delay
+function delay(millis) {
```

Comment:
I believe this function is declared at many places already, do you plan to reuse it everywhere?

Reply:
will check if i can consolidate them all here.